schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                            "southBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            },
+                            "northBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            }
+                        },
+                        "required": [
+                            "westBoundLongitude",
+                            "eastBoundLongitude",
+                            "southBoundLatitude",
+                            "northBoundLatitude"
+                        ]
+                    },
+                    "geoLocationPolygons": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "polygonPoints": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/geoLocationPoint"
+                                    },
+                                    "minItems": 4
+                                },
+                                "inPolygonPoint": {
+                                    "$ref": "#/definitions/geoLocationPoint"
+                                }
+                            },
+                            "required": [
+                                "polygonPoints"
+                            ]
+                        },
+                        "uniqueItems": true
+                    }
+                }
+            },
+            "uniqueItems": true
+        },
+        "provider": {
+            "description": "The provider of the resource. This can be a DOI registration agency or a repository.",
+            "type": "string",
+            "enum": [
+                "Crossref",
+                "DataCite",
+                "GitHub",
+                "JaLC",
+                "KISTI",
+                "mEDRA",
+                "OP"
+            ]
+        },
+        "alternate_identifiers": {
+            "description": "Alternate identifiers for the resource.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "alternateIdentifier": {
+                        "type": "string"
+                    },
+                    "alternateIdentifierType": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "alternateIdentifier",
+                    "alternateIdentifierType"
+                ]
+            }
+        },
+        "content_url": {
+            "description": "The URL of the content of the resource.",
+            "type": "array",
+            "items": {
+                "type": "string",
+                "format": "uri"
+            }
+        },
```

Comment:
As mentioned above, could do with some clarification of the difference between `id`, `url`, and `content_url`.

Reply:
`content_url` probably is clearer in the latest `commonmeta-py` schema with the new `files` property. Essential this is metadata describing the `DataDownload`, including the URL of a file to download (`content_url`), checksum, MIME type and file size.

Reply:
Yup, that makes sense now -- the examples I've been looking at (which use the Frictionless schema) call it the `path`.

Reply:
I will look into the frictionless schema and compare.

Reply:
And more documentation of id vs.url would be good. In commonmeta all ids should ideally be urls, with the special case for CURIEs.